applications/GeoMechanicsApplication/tests/cpp_tests/test_utilities/model_setup_utilities.h

```diff
@@ -13,6 +13,7 @@
 #pragma once
 
 #include "geo_aliases.h"
+#include "includes/properties.h"
```

Comment:
This include we might be able to move to the cpp file.

Reply:
You are right, done.